.eslintrc

```diff
@@ -4,9 +4,12 @@ rules:
     - unix
   no-console:
     - 1
+  max-len: "off" # Use prettier default line length
```

Comment:
See https://github.com/prettier/prettier-eslint/issues/55.

Reply:
Isn't that fixed though?

Reply:
You'd think, right?

Without this I get lint errors from prettier that, when fixed, result in lines longer than 120 characters. I'm not entirely sure why. It certainly doesn't feel like the worry-free future that prettier promised.